libvast/src/system/indexer.cpp

```diff
@@ -108,16 +104,11 @@ active_indexer(active_indexer_actor::stateful_pointer<indexer_state> self,
 
 indexer_actor::behavior_type
 passive_indexer(indexer_actor::stateful_pointer<indexer_state> self,
-                uuid partition_id, value_index_ptr idx) {
-  if (!idx) {
-    VAST_ERROR("{} got invalid value index pointer", *self);
-    self->quit(caf::make_error(ec::end_of_input, "invalid value index "
-                                                 "pointer"));
-    return indexer_actor::behavior_type::make_empty_behavior();
-  }
-  self->state.name = fmt::format("indexer-{}", idx->type());
+                uuid partition_id, value_index_ptr index) {
+  VAST_ASSERT_CHEAP(index);
+  self->state.name = fmt::format("indexer-{}", index->type());
```

Comment:
If we're touching this anyways, does it make sense to proactively switch to a static name to reduce the scope of the CAF 0.18 refactoring? Or would it just create additional merge conflicts? (cc/ @patszt )

Reply:
I think either change won;t impact refactoring

Reply:
I'll leave this as-is then.